src/frontend/src/btc/components/convert/BtcConvertTokenWizard.svelte

```diff
@@ -52,6 +52,9 @@
 
 	const progress = (step: ProgressStepsConvert) => (convertProgressStep = step);
 
+	let insufficientFunds: boolean;
```

Comment:
Do we need to pass two property up and down? Or one would fit? I guess depends where the logic finds place best?

Reply:
Just one prop does the job too. Updated.